web_src/js/features/comp/ComboMarkdownEditor.js

```diff
@@ -130,7 +130,7 @@ class ComboMarkdownEditor {
       } else if (key === '@') {
         const matches = [];
         for (const obj of window.config.tributeValues) {
-          if (obj.key.includes(text)) {
+          if (obj.key.toLowerCase().includes(text.toLowerCase())) {
```

Comment:
Could the `text.toLowerCase()` be a variable (out of the `for`) to slightly improve performance?